queries/buildings.jinja2

```diff
@@ -73,6 +79,8 @@ SELECT
     NULL AS roof_orientation,
     "addr:housenumber" AS addr_housenumber,
     "addr:street" AS addr_street,
+    tags->'addr:floor' AS addr_floor,
+    tags->'level' as level,
```

Comment:
Taginfo suggests that only [0.02% of nodes](http://taginfo.openstreetmap.org/keys/addr%3Ahousenumber?filter=nodes#combinations) tagged with `addr:housenumber` also have `level` set. Were there some specific nodes you were looking at where `level` was important?